### DISCUSSION TITLE
Tutorial 1: Basic System Administration after logging into your head node

### DISCUSSION THREAD

**Discussion by nyameko** [chpc-tech-eval/scc#49] [Show and tell]
The following provides sample output from the commands you ran in the system administration sections:

<img width="1413" alt="man_sudo_feedback" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/57ce6444-0866-4b72-a6cb-7def9b247dd1">

**Comment by nyameko** [comment]
<img width="1427" alt="sudo_with-h_flag" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/16fc2f43-f9e5-46a1-b71d-1964a5945814">

**Reply by nyameko** [reply]
```bash
   # Check the help pages of a command using the `--help or -h` switch.
   sudo -h
   ifconfig -h
   ping -h
   ssh-keygen --help
   grep --help
```

**Reply by nyameko** [reply]
You can explore `man` manual pages using, for example:

```bash
# Review the man pages of a number of different Linux utilities.
# Remember you can exit the man page documentation by pressing  `q`.
man sudo
man ping
man tracepath
man ls
man cat
   ```

**Reply by n-zwane** [reply]
Team Technocrats SCC 2025
<img width="944" height="1072" alt="basics" src="https://github.com/user-attachments/assets/071a3fc1-6c98-4be3-a1a7-6d254760370b" />

**Comment by nyameko** [comment]
<img width="602" alt="search_commands_grep" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/6275d7d1-acd1-48ac-b095-47e7eae622c6">

**Comment by nyameko** [comment]
<img width="630" alt="history_command" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/86d1be79-432b-488b-9b02-aeb0d07e200c">

**Comment by nyameko** [comment]
<img width="1440" alt="rerun_command_from_history" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/f7bf3d46-2720-4183-8154-088a04af0107">

**Comment by prince-taker** [comment]
![ctrl alt elite man sudo](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/15ced19a-92b8-4ea8-aced-e3ce9ed8c79f)

Ctrl alt elite man sudo screenshot

**Reply by prince-taker** [reply]
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/2dba374f-c457-46fb-94c8-f399586f3cf7)

**Comment by iamyou-uoymai** [comment]
![Q A](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/120018440/db09816b-bf94-4952-b3fe-5ffd65510b31)

**Comment by Dc-Spears** [comment]
CPUT (DC Spears)

![cput-sudo-h-2](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175037009/e0baabc8-1fd4-49f5-9832-a721323c327d)


We like to use the (dash h) '-h' instead of 'man' more often as it is easier to navigate. This is most probably our favourite flag.

**Comment by Cyphers-UFH** [comment]
UFH (Cyphers)

![Screenshot 2024-07-09 053221](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175048513/6d748126-f3af-4e3b-a630-0a07aaf562e6)
![Screenshot 2024-07-09 054353](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175048513/cbcbe75e-bea2-4124-ae47-2989dbc36763)
![Screenshot 2024-07-09 054509](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175048513/c46853c3-8335-49ee-9821-f55cf36f9272)
![Screenshot 2024-07-09 054545](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175048513/74fb0801-ae45-47b5-af92-3c133fb30eb0)

**Comment by Talhahpatelia** [comment]
## Wits-A
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/fccb22da-9f71-43c2-a1fc-fa264dcb4896)

**Comment by Talhahpatelia** [comment]
### Wits-A
<img width="960" alt="image" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/91e7d934-610f-4b0f-968f-fc3cec5bbb03">

**Comment by CraftierLamb599** [comment]
# Wits-1
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/137203651/9576aec0-6273-48e3-b6ad-e66028b182fd)

**Comment by SfisoMncube** [comment]
Khulisa IP-A

![Screenshot (237)](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/151764841/709bb191-5625-4138-99e1-651982087f47)

**Comment by ADELAIDEMOGALE** [comment]
### **Khulisa Data**

![Screenshot 2024-07-10 182920](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/166052302/bd5a911d-8c89-4d69-9f2a-96d072709668)

![Screenshot (691) (1)](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/166052302/c9d71770-52a4-40d5-aea0-c4d9c20d3a77)

![Screenshot (690)](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/166052302/0441881e-8882-47f6-a759-1da4034f7e68)

**Comment by Rosemary2301** [comment]
![ADD_TO_FILE SCREENSHOT](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/3c3b9a3e-d21c-4cb3-99ca-207f75c60c32)

**Comment by Rosemary2301** [comment]
Algorithm Avengers
![Sudo h](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/a85edae4-7104-4391-8270-474d6ea5b8cf)

**Comment by Rosemary2301** [comment]
Algorithm Avengers
![history command](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/ce9f8ad6-ecd2-49c7-9f70-5b3a11fcaf4b)

**Comment by lisabeeaan** [comment]
**HPC Wizards**
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/136716760/95f3490b-bfbf-44f3-84ff-94cf8431809a)
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/136716760/9ebd3418-8886-4ebc-9995-7e0a2cdbf73e)

**Comment by SizweMarole-21389** [comment]
Cybershield 
![commands](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175023186/aa992d95-4e15-4df6-a3dc-a644b4c6bef3)

**Comment by hoperiley0806** [comment]
**Parallel Pioneers**

![WhatsApp Image 2024-07-12 at 02 47 19_27d1eb6e](https://github.com/user-attachments/assets/dce315a5-af39-4c1f-9c99-925fe96f06c6)

**Comment by 15kay** [comment]
HYPERNET

![image](https://github.com/user-attachments/assets/7f3fcd9b-41a4-479c-a972-f1e251bbf951)

**Comment by Kohzmik** [comment]
CUT_Team
![image](https://github.com/user-attachments/assets/c29fea83-2947-4387-bf79-6b6ad12fd1e6)
![image](https://github.com/user-attachments/assets/5a5428d5-2e72-42f3-8638-00c44cd45efd)
![image](https://github.com/user-attachments/assets/658b267f-6948-4b1b-ac71-0c5839fe6530)

**Comment by AnnaLukose** [comment]
**EEElite (UCT)**
![1](https://github.com/user-attachments/assets/9a7f34a6-ae79-43ac-8b2e-02af0df33a00)
![2](https://github.com/user-attachments/assets/09515f60-5a53-4cb4-aedd-d48475d1379a)

**Comment by ngqolsss** [comment]
WSU ELITE
![Screenshot 2024-07-09 165424](https://github.com/user-attachments/assets/4b191530-8d71-4cb2-a230-e56bc48fe0cc)

**Comment by 15kay** [comment]
HYPERNET

[image: image.png]

**Comment by Iloke-Alusala** [comment]
0V1RCL0CK1D

![Image 2025-07-07 at 16 45](https://github.com/user-attachments/assets/7bb72133-02eb-4f37-aaca-8866a8c21ff9)

**Comment by MphephuSamuel** [comment]
computer champions 

![image](https://github.com/user-attachments/assets/ef1a9735-f941-4496-bb3e-593a08a067ec)

**Comment by DCspearsv2** [comment]
DC Spears
![tut1 3](https://github.com/user-attachments/assets/f78bb350-99d1-4143-aeb4-e48edd10aa93)

**Comment by UNISA-SCT25** [comment]
![Tut1_AfterHNLogin](https://github.com/user-attachments/assets/3e793046-c517-4f57-bdd4-b170cfe5a558)

**Comment by MubeenD1** [comment]
Wits-1
![image](https://github.com/user-attachments/assets/edf0501d-eced-4855-93eb-78fb20034c40)

![image](https://github.com/user-attachments/assets/17ed4e48-3ad9-4c53-a632-b9d5ab5ba337)

![image](https://github.com/user-attachments/assets/b85e2fa7-967c-49fe-bbb2-65d38fce3911)

**Comment by ThapeloVundla** [comment]
<img width="830" height="997" alt="Screenshot from 2026-07-06 14-46-50" src="https://github.com/user-attachments/assets/a20dad79-534a-4503-9d01-2d54cf91028b" />

DC Spears

**Comment by CadmusKei** [comment]
Team Jargon (UWC) |  SCC 2026 | Further Basic Linux Commands
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/d4d253b2-cb6f-42c5-874f-a4691d8e80db" />

**Reply by CadmusKei** [reply]
Team Jargon (UWC) | SCC 2026 | `sudo -h` Example | 
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/6ed323b5-ca6e-4d66-8737-f66021693011" />

**Comment by saxs-14** [comment]
3ncryp+3d (UMP)  sudo -h command
<img width="1918" height="1078" alt="Sudo -h" src="https://github.com/user-attachments/assets/23ff48c2-8aec-441d-be0e-94015ff1fdce" />

**Comment by saxs-14** [comment]
3ncryp+3d (UMP) histoty and !14 commands
<img width="1917" height="1077" alt="history + !14" src="https://github.com/user-attachments/assets/886dcb13-8437-42bd-a958-81992512162f" />

**Comment by rsabreadking** [comment]
WITS-1 basic commands

<img width="650" height="436" alt="image" src="https://github.com/user-attachments/assets/24d5231a-1129-4ff7-92f3-fb7fa9199c1e" />

**Comment by omo-bit** [comment]
Wits -1
sudo journalctl

<img width="1457" height="623" alt="image" src="https://github.com/user-attachments/assets/1cb5ab36-9af6-4d6f-be47-027c6ea4a233" />

**Comment by ntando-memela** [comment]
WITS1-2026

<img width="1860" height="954" alt="Screenshot from 2026-07-06 15-52-13" src="https://github.com/user-attachments/assets/743fc1c5-7b77-4562-acea-f6e8ee4960d1" />
<img width="1860" height="954" alt="image" src="https://github.com/user-attachments/assets/75a4ecd3-73f9-425f-8e54-6ea7e96c1529" />

**Comment by omo-bit** [comment]
Wits -1
sudo systemctl status systemd-networkd

<img width="1310" height="181" alt="image" src="https://github.com/user-attachments/assets/bbeba2cd-8816-4bc3-84e0-b4b330692264" />

**Comment by RoarkeBailey** [comment]
<img width="761" height="255" alt="Screenshot from 2026-07-06 14-39-41" src="https://github.com/user-attachments/assets/48cc85d8-198d-4454-97a3-fe729e930e16" />

G3MIKNIGHTS example of basic commands

**Comment by UNISA-QC-QUEENS** [comment]
UNISA_QC_QUEENS
<img width="645" height="341" alt="Screenshot from 2026-07-06 16-31-43" src="https://github.com/user-attachments/assets/935d1729-b902-4fb8-b008-d58ade6a33bf" />

**Comment by K659** [comment]
Resilient Geeks (UL)
<img width="1811" height="1010" alt="image" src="https://github.com/user-attachments/assets/52d8c15a-9798-4b50-a810-f8fb74cbb9f2" />

**Comment by LufunoMiles** [comment]
Digital dynamos(University of venda)        <img width="1919" height="1070" alt="Screenshot from 2026-07-06 17-41-26" src="https://github.com/user-attachments/assets/7a9ad02c-06b5-44b3-a745-048bc83f9482" />

**Reply by 23026641** [reply]
Digital Dynamos(University of venda) : 
<img width="1416" height="1050" alt="WhatsApp Image 2026-07-09 at 16 35 48" src="https://github.com/user-attachments/assets/fcd3d071-9b56-4307-a6ec-e2d998a23308" />

**Comment by bytedebuggers-dotcom** [comment]
**Basic system administration**
<img width="1299" height="262" alt="Screenshot From 2026-07-06 22-35-47" src="https://github.com/user-attachments/assets/96cfe7c8-b12c-4936-9c60-c757bf819dd6" />

**sudo manual**

<img width="1302" height="690" alt="Screenshot From 2026-07-06 22-38-55" src="https://github.com/user-attachments/assets/980e20c6-7d54-4b8b-85a4-db88f0efc5bc" />

**Comment by Leo-OB** [comment]
## DC Axes (CPUT) | SCC 2026
### Outputs of basic system administration commands `man sudo` to list command info, creation of a file using `touch`, appending to file using `echo` and `>>`, `cat` and `grep` to filter viewed contents of file, `sudo -h` listing useful available commands:
<img width="1920" height="1080" alt="man sudo" src="https://github.com/user-attachments/assets/be381477-5112-427d-bed5-ed9ebd61e5cf" />

<img width="1005" height="524" alt="image" src="https://github.com/user-attachments/assets/2a011f2d-4837-4ca8-aa5e-2a90a585facd" />

<img width="1920" height="1080" alt="sudo -h" src="https://github.com/user-attachments/assets/00e65ba7-0627-4558-bad0-651f32633844" />

**Comment by switchsquad** [comment]
<img width="1918" height="1017" alt="image" src="https://github.com/user-attachments/assets/f8234782-dda4-4057-abcf-01ad5aaae263" />

Switch Squad (TUT)

**Comment by JayJay-lab** [comment]
Sol_Plaatje_A SCC 2026          
<img width="1901" height="412" alt="image" src="https://github.com/user-attachments/assets/fed79c2a-802f-445e-98ea-a7f3aa4e2218" />

**Comment by mpho21925** [comment]
Ampere Alliance | SCC 2026
using the --help command to get more information about the sudo

<img width="1920" height="1080" alt="Screenshot 2026-07-08 084822" src="https://github.com/user-attachments/assets/301958fb-b3ca-4729-b18c-5e84a1b0660d" />

**Comment by 111nation** [comment]
**UPtime**
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/1158cf2c-e7f0-40af-85d3-0dd9f2b9f5c0" />

**Comment by Lihle0512** [comment]
ChaosCompilers

<img width="1852" height="966" alt="Screenshot from 2026-07-09 15-36-49" src="https://github.com/user-attachments/assets/7e2e9697-247a-4d0d-9a63-c58c832cad9a" />
<img width="1852" height="966" alt="Screenshot from 2026-07-09 15-37-33" src="https://github.com/user-attachments/assets/3edd7e15-86df-467d-a63a-20721005c3f0" />
<img width="1862" height="188" alt="Screenshot from 2026-07-09 15-41-52" src="https://github.com/user-attachments/assets/3963bd83-a54a-43c5-b61d-af08b4995b6f" />

**Comment by majaha8** [comment]
Team Nebula|2026
<img width="935" height="1011" alt="basic admin" src="https://github.com/user-attachments/assets/cf1f60b9-412f-4a93-93e4-5f8881899ffe" />

**Comment by saxs-14** [comment]
3ncrypt3d (UMP) 
<img width="1912" height="1078" alt="image" src="https://github.com/user-attachments/assets/414f4847-60c1-4cd4-85e7-7e2b5e6dafc2" />
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/7c582dc0-119e-4328-8833-65317bf24f78" />
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/a583921b-6b86-41d6-8a3a-6cb7ac16b3f2" />
<img width="1918" height="1073" alt="image" src="https://github.com/user-attachments/assets/b15b0a57-9896-4cae-b7ec-3ee66c07fe53" />
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/8c6ef861-5e12-4ac5-b5af-d5084792e1f8" />
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/f5b52ff5-f332-4084-bacc-a04f0ed40032" />
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/f02d8500-f9e6-47fa-bee5-a3b097c4e84b" />
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/b28d3391-d221-4153-870a-76ab965151e8" />
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/5a52333d-fc9b-4e5e-b5f2-a034b2b76b5f" />
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/4711b8b3-3aac-430e-a213-800ce7921da4" />

**Comment by BleSSednav3** [comment]
Resilient geeks
<img width="1530" height="1036" alt="image" src="https://github.com/user-attachments/assets/9f6903ad-304d-4489-bbb0-9ddb17cf59f8" />
<img width="1530" height="1036" alt="image" src="https://github.com/user-attachments/assets/6bf46290-9cbb-4c6c-90b2-1f1908ad7d67" />

**Comment by Leeu-Fusi** [comment]
Cluster Commanders | CUT
<img width="1195" height="976" alt="Screenshot from 2026-07-10 12-23-04" src="https://github.com/user-attachments/assets/8eba266e-b124-475a-aff7-fb0ca9205e24" />
<img width="1195" height="976" alt="Screenshot from 2026-07-10 12-23-22" src="https://github.com/user-attachments/assets/521981db-4392-480a-a320-dfe82506f8b8" />
<img width="1195" height="1002" alt="Screenshot from 2026-07-10 12-20-57" src="https://github.com/user-attachments/assets/4cd307e6-3055-471d-bba4-b880ebad2b82" />
<img width="1195" height="1002" alt="Screenshot from 2026-07-10 12-21-12" src="https://github.com/user-attachments/assets/f7410efa-b17a-4f79-a801-913b55e5d9fb" />
<img width="1195" height="787" alt="Screenshot from 2026-07-10 12-22-39" src="https://github.com/user-attachments/assets/59ef0ab0-086c-47d9-b89a-b90a8843ce12" />

**Comment by MrLampies** [comment]
DC Spears

<img width="1014" height="413" alt="Screenshot from 2026-07-07 11-57-34" src="https://github.com/user-attachments/assets/9b41fe8a-3497-454f-afa2-244fc2dc8516" />
<img width="989" height="565" alt="Screenshot from 2026-07-07 11-58-32" src="https://github.com/user-attachments/assets/1f0861b6-6c14-4c84-a051-d155e4bfb949" />
<img width="989" height="564" alt="Screenshot from 2026-07-07 15-30-34" src="https://github.com/user-attachments/assets/2a2d5cbe-2038-4ce1-9d49-b61a609f4500" />

**Comment by Slindokuhle19** [comment]
UnizuluComputeX
<img width="863" height="955" alt="Screenshot from 2026-07-10 22-04-39" src="https://github.com/user-attachments/assets/e69d67b9-66b8-496f-afb2-4ac961768f20" />

<img width="878" height="961" alt="Screenshot from 2026-07-10 22-05-09" src="https://github.com/user-attachments/assets/c548ab2c-6ffe-494b-9438-644fef6b44b8" />
